flink-connector-examples/build.gradle

```diff
@@ -21,22 +21,19 @@ apply plugin: 'eclipse'
 sourceCompatibility = "1.8"
 archivesBaseName = 'pravega-flink-examples'
 
-ext {
-    scalaJava8CompatVersion = '0.7.0'
-}
-
 dependencies {
-    compile "org.scala-lang.modules:scala-java8-compat_2.11:${scalaJava8CompatVersion}"
-    compile "io.pravega:pravega-connectors-flink_2.11:${flinkConnectorVersion}"
-    compile "org.apache.flink:flink-streaming-java_2.11:${flinkVersion}"
-    compile "org.apache.flink:flink-streaming-scala_2.11:${flinkVersion}"
+    compile "de.javakaffee:kryo-serializers:$kryoSerializerVersion"
+    compile "org.scala-lang.modules:scala-java8-compat_$flinkScalaVersion:${scalaJava8CompatVersion}"
+    compile "io.pravega:pravega-connectors-flink_$flinkScalaVersion:${flinkConnectorVersion}"
+    compile "org.apache.flink:flink-streaming-java_$flinkScalaVersion:${flinkVersion}"
+    compile "org.apache.flink:flink-streaming-scala_$flinkScalaVersion:${flinkVersion}"
     compile "org.slf4j:slf4j-log4j12:1.7.25"
```

Comment:
Could we also use a `version` variable for slf4j?

Reply:
fixed